package.json

```diff
@@ -45,11 +46,6 @@
     "gulp-nsp": "^2.1.0"
   },
   "repository": "mateusmedeiros/generator-prelude",
-  "scripts": {
-    "link": "npm run prepublish && sudo npm link",
-    "prepublish": "cp -R generators generatorsSrc && gulp publish",
```

Comment:
Hmm, but I was running `npm link` and the generator without any problems, it seem's like babel is not being required.